### ISSUE TITLE
wrong! there is no option to install Visual Studio for Mac

### ISSUE THREAD

**Issue by klauskobald** [MicrosoftDocs/visualstudio-docs#5329]
Autocompletion or jumping to classes from within the code does not work anymore :-(

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0be31e5a-ccb8-93c5-43a0-ef8327f4ed4e
* Version Independent ID: 2ed7a14e-12ea-ea72-6678-0db050023a62
* Content: [Setup Visual Studio for Mac Tools for Unity - Visual Studio 2019 for Mac](https://docs.microsoft.com/en-us/visualstudio/mac/setup-vsmac-tools-unity?view=vsmac-2019#feedback)
* Content Source: [mac/setup-vsmac-tools-unity.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/mac/setup-vsmac-tools-unity.md)
* Product: **visual-studio-mac**
* Technology: **vs-ide-general**
* GitHub Login: @therealjohn
* Microsoft Alias: **johmil**

**Comment by therealjohn** [comment]
@klauskobald Which version of Unity were you installing that didn't show Visual Studio for Mac? Did you already have Visual Studio for Mac installed on the Mac? 

> Autocompletion or jumping to classes from within the code does not work anymore

Sounds like a separate issue. Please Report a Problem on Developer Community with the details of the issue. Or, use the Help > Report a Problem menu in VS for Mac. 

https://developercommunity.visualstudio.com/spaces/41/index.html

**Comment by indiesaudi** [comment]
We just updated the windows doc to reflect the changes that Unity has done to the installation process, we will continue to update the docs for VS For Mac too.

**Comment by klauskobald** [comment]
hi

> Am 19.05.2020 um 13:34 schrieb John Miller <notifications@github.com>:
> 
> 
> @klauskobald <https://github.com/klauskobald> Which version of Unity were you installing that didn't show Visual Studio for Mac? Did you already have Visual Studio for Mac installed on the Mac?
> 
> 
yes, I downloaded it last week because I found out that it did not update itself since a year and debugger stopped working.

I am using the latest unity hub - anyway. 
The real problem is that automcompleten is not working stable anymore.
I thought that maybe that unity plugin is missing, but it is there.

There is folders in the solution that are empty. But they are not! When I open a file from unity it suddenly shows up. But cmd-d is not working. It´s not opening the selected symbol.
It does not know namespaces or unity classes. But sometimes it does.

I cleared the cache - I cleared the project files. Nothing helps. Is there maybe an older version I can download?

or what else can I do to understand / reload all C# files?
